packages/lab/stories/file-drop-zone/file-drop-zone.stories.tsx

```diff
@@ -348,61 +181,57 @@ interface ResultCardProps {
   result: ResultCardType | undefined;
 }
 
-export const ResultCard = ({ result }: ResultCardProps) => {
+export const Results = ({ result }: ResultCardProps) => {
```

Comment:
Is Results, intended to be a story?

Reply:
it was there before so i left it, but happy to remove it